### DISCUSSION TITLE
"Permanent mud" as a selectable surface (`surface=mud`)

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#4300] [Ideas]
I am experimenting with surface overlay while I am on holidays and thought about supporting as many real surfaces as possible, including `surface=mud`

Options:

- do not support it in either overlay or as a selectable surface (ways tagged as `surface=mud` will not be editable with surface overlay)
- support in surface overlay, do not offer it as a selectable option in answer quests
- support in both

My main worry is that it seems likely that people may select it mistakenly if mapping jut after rain and that it can increase translator work more than benefit from enabling it.

Right now I think about supporting it in surface overlay and to not offer it as an option is surface quests.\

----------------------

Was added.

**Comment by mnalis** [comment]
It makes sense to me to only enable it in overlay, as long as it is very clearly named so there is no confusion. 
_"Year-round mud"_ might be more clear and more easily translatable (at least in Croatian).
